rollup.config.js

```diff
@@ -0,0 +1,40 @@
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+import { readFileSync } from 'node:fs';
+import alias from '@rollup/plugin-alias';
+
+const packageJson = JSON.parse(
+	readFileSync(new URL('./package.json', import.meta.url))
+);
+
+const __dirname = fileURLToPath(new URL('.', import.meta.url));
+const projectRootDir = path.resolve(__dirname);
+const assetsDir = path.resolve(projectRootDir, 'app/javascript');
+
+
+/**
+ * @type {import('rollup').RollupOptions}
+ */
+export default [
+	// build dist folder with all files from app/javascript using relative imports.
+	// let bundler tools like webpack or rollup to process our package
+	{
+		input: ['app/javascript/active_admin.js'],
+		output: {
+			format: 'es',
+			dir: 'dist',
+			preserveModules: true,
+		},
+		external: Object.keys(packageJson.dependencies),
+		plugins: [
+			alias({
+				entries: [
+					{
+						find: 'active_admin',
+						replacement: path.join(assetsDir, 'active_admin'),
+					},
+				]
+			})
+		]
+	}
+];
```

Comment:
indentation here seems to use tabs instead of spaces

Reply:
Yes should be spaces, sorry. Of course this is always the first thing I'd bring up too. 🙈

Reply:
Fixed